modules/serverless-pingsource-yaml.adoc

```diff
@@ -0,0 +1,144 @@
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+
+[NOTE]
+====
+The following procedure requires you to create YAML files.
+
+If you change the names of the YAML files from those used in the examples, you must ensure that you also update the corresponding CLI commands.
+====
+
+.Procedure
+
+. To verify that `PingSource` is working, create a simple Knative
+service that dumps incoming messages to its log.
+.. Copy the example YAML into a file named `service.yaml`:
++
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+----
+.. Create the service:
++
+----
+$ oc apply --filename service.yaml
+----
+
+. For each set of ping events that you want to request, create a PingSource in the same namespace as the event consumer.
+.. Copy the example YAML into a file named `ping-source.yaml`:
++
+----
+apiVersion: sources.knative.dev/v1alpha2
+kind: PingSource
+metadata:
+  name: test-ping-source
+spec:
+  schedule: "*/2 * * * *"
+  jsonData: '{"message": "Hello world!"}'
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+----
+.. Create the PingSource:
++
+----
+$ oc apply --filename ping-source.yaml
+----
+
+. Check that the controller is mapped correctly by entering the following command and inspecting the output:
++
+----
+$ oc get pingsource.sources.knative.dev test-ping-source -oyaml
+----
++
+The output should be similar to:
++
+----
+apiVersion: sources.knative.dev/v1alpha2
+kind: PingSource
+metadata:
+  annotations:
+    sources.knative.dev/creator: developer
+    sources.knative.dev/lastModifier: developer
+  creationTimestamp: "2020-04-07T16:11:14Z"
+  generation: 1
+  name: test-ping-source
+  namespace: default
+  resourceVersion: "55257"
+  selfLink: /apis/sources.knative.dev/v1alpha2/namespaces/default/pingsources/test-ping-source
+  uid: 3d80d50b-f8c7-4c1b-99f7-3ec00e0a8164
+spec:
+  jsonData: '{ value: "hello" }'
+  schedule: '*/2 * * * *'
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+      namespace: default
+----
+
+.Verfication steps
+
+You can verify that the Kubernetes events were sent to the Knative event sink by looking at the message dumper function logs.
+By default, a Knative service will be created to receive each new message that is sent. The service will be terminated after 60 seconds.
+The example shown in this guide creates a PingSource that sends a message every 2 minutes, therefore each message that is sent will be received by a newly created pod, which will be terminated after 60 seconds.
+
+. Watch for new pods created:
++
+----
+$ watch oc get pods
+----
+
+. Cancel watching the pods using Ctrl+C, then look at the logs of the created pod:
++
+----
+$ oc get pods
```

Comment:
Now, after changes this first command `oc get pods` is no longer required, as we already checked that pod is there.